webpack_loader/loader.py

```diff
@@ -39,15 +39,23 @@ def get_assets(self):
         return self.load_assets()
 
     def filter_chunks(self, chunks):
```

Comment:
Did you check all calls to `filter_chunks`?

Reply:
Also, could you please track this change of behavior in a Unreleased section of the CHANGELOG?

Reply:
Yep, there is only one call at https://github.com/django-webpack/django-webpack-loader/blob/cc8c0a0a192ccfba8cc079c33f2a5f89c11b4822/webpack_loader/loader.py#L92-L102